src/main/webapp/WEB-INF/tags/student/feedbackResults/feedbackSessionDetailsPanel.tag

```diff
@@ -0,0 +1,41 @@
+<%@ tag description="studentFeedbackResults.jsp - Displays feedback session details" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib tagdir="/WEB-INF/tags/student/feedbackResults" prefix="feedbackResults" %>
+<%@ attribute name="feedbackSession" type="teammates.common.datatransfer.FeedbackSessionAttributes" required="true" %>
+
+<div class="well well-plain">
+    <div class="panel-body">
+        <div class="form-horizontal">
+            <div class="panel-heading">         
+                <feedbackResults:displayFeedbackSessionInfo>
+                    <jsp:attribute name="heading">Course ID:</jsp:attribute>
+                    <jsp:body>
+                        <c:out value="${feedbackSession.courseId}"/>
+                    </jsp:body>
+                </feedbackResults:displayFeedbackSessionInfo>
```

Comment:
If `heading` is just a `String`, maybe this would work better:
````
<feedbackResults:displayFeedbackSessionInfo heading="Course ID:">
    <c:out value="${feedbackSession.courseId}"/>
</feedbackResults:displayFeedbackSessionInfo>
````
Rather than treating it as a fragment.